azero/contracts/most/lib.rs

```diff
@@ -374,6 +377,79 @@ pub mod most {
             Ok(())
         }
 
+        /// Invoke this tx to initiate weth -> ether transfer to the destination chain.
+        ///
+        /// Upon checking basic conditions the contract will burn the `amount` number of weth tokens from the caller
+        /// and emit an event which is to be picked up & acted on up by the bridge guardians.
+        #[ink(message, payable)]
+        pub fn send_request_native_ether(
+            &mut self,
+            amount: u128,
+            dest_receiver_address: [u8; 32],
+        ) -> Result<(), MostError> {
+            self.ensure_not_halted()?;
+
+            let mut data = self.data()?;
+
+            let src_token_address = data.weth.ok_or(MostError::UnsupportedPair)?;
```

Comment:
I think we may want to add a custom error here, not just `UnsupportedPair`, which is inaccurate

Reply:
I think `UnsupportedPair` is not entirely inaccurate in the context of `send_request_native_ether(...)` as it suggests that the "pair" ether(azero) -> ether(eth) has not been set (just like in the context of `send_requst(...)` for regular pairs).
However, to be more explicit we can replace it with a custom error like `WrappedEthNotSet`.

Reply:
Yes, I agree that the issue is somewhat similar to the one typically indicated by UnsuppoetedPair, but I think it's different enough to get it's own error variant